src/v35.js

```diff
@@ -1,12 +1,29 @@
 import bytesToUuid from './bytesToUuid.js';
+import validate from './validate.js';
+
+// Int32 to 4 bytes
+function numberToBytes(num, bytes) {
+  const offset = bytes.length;
+  bytes.push(0, 0, 0, 0);
+
+  for (let i = 0; i < 4; ++i) {
+    const byte = num & 0xff;
+    bytes[offset + 3 - i] = byte;
```

Comment:
I think this line could warrant a comment. Something like fill the 4 appended bytes right-to-left.

Reply:
Or a link to the respective stackoverflow question 😉